css/main.css

```diff
@@ -0,0 +1,433 @@
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+/* =====contact me section====  */
+
+.contact {
+  background-color: #fff;
+  padding-top: 80px;
+  min-height: 779px;
+}
+
+.contact-container {
+  background-color: #fffdfc;
+  min-height: 779px;
+  border-top-left-radius: 150px;
+  object-fit: cover;
+}
+
+.contact-container h2 {
+  padding-top: 103px;
+  color: #ffff;
+  text-align: center;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.contact-P {
+  padding-top: 12px;
+  padding-left: 10px;
+  padding-right: 10px;
+  color: #172b4d;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
```

Comment:
- [ ] It is advisable to stick to the design from Figma as much as possible. Because having a project that is aesthetically pleasant would increase your chance of landing a job, because it would inspire confidence in your ability 🥸 So, please fix your paragraph so it would looks like the provided design. Below are screenshots to refer to.

Your design 👇
![Screenshot from 2023-05-24 00-39-52](https://github.com/Cabraham1/contact-form-trial/assets/10439283/8d3a1f42-2919-4430-89e6-feef2093b9dd)

Design from Figma 👇
![Screenshot from 2023-05-24 00-39-18](https://github.com/Cabraham1/contact-form-trial/assets/10439283/29be791e-843c-4d80-b0dc-807491e1fea8)